CHANGELOG.md

```diff
@@ -48,6 +52,14 @@ Users/clients that do not make use of these endpoints will not be impacted.
   - Added `max_download_time` column to PostgreSQL `providers` table
   - Updated `@cumulus/ingest/lock` to check expired locks based on `provider.maxDownloadTime`
 
+### Fixed
+
+- **CUMULUS-3427**
+  - fixed issue where some lambda and task memory sizes and timeouts were not configurable
+  - changed the naming conventions for memory size and timeouts configuration to simply the lambda name
```

Comment:
do you mean 'simplify'?

Reply:
Ah yeah must be a bad merge, let me fix it 👀

Reply:
Actually looks like it was included in the latest release so that entry is fine where it is @charleshuang80 @Nnaga1 https://github.com/nasa/cumulus/pull/3502

Reply:
https://github.com/nasa/cumulus/blob/b58c0051f05a5234bd9cff93390ae54efdeddd24/CHANGELOG.md?plain=1#L55-L61 
https://github.com/nasa/cumulus/blob/b58c0051f05a5234bd9cff93390ae54efdeddd24/CHANGELOG.md?plain=1#L93-L97
Looks like that entry got duplicated somehow, so I fixed it 
https://github.com/nasa/cumulus/pull/3504/commits/57d19379baca95dc9c436ff9ef89d148279a9ec6#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL55-L62

@charleshuang80 @Nnaga1